docs/plugin/interfaces/tags.md

```diff
@@ -0,0 +1,483 @@
+When designing a COM-RPC interface, it is possible to add specific comments (known as annotations or tags) that will drive the code generation. 
+
+All tags follow the same format of `@` followed by the tag name. They can be inserted into the interface using either inline `//` comments or `/* */` block comments. Tags can influence the generation of the COM-RPC ProxyStubs and the generated JSON-RPC interfaces. They can also be used to configure the documentation generation
+
+## Summary
+
+### General purpose tags
+
+| Tag|Short Description|Deprecated|StubGen|JsonGen|Scope|
+|--|--|:--:|:--:|:--:|:--:|
+|[@stubgen:skip](#stubgenskip)|Stop processing current file, now deprecated| Yes | Yes|Yes|File|
+|[@stop](#stop)|Same as @stubgen:skip, not deprecated, but also never advertised ||Yes| Yes|File|
+|[@stubgen:omit](#stubgenomit)|Omit processing of the next class or method | | Yes| No (but has side-effects)| Class, Method|
+|[@omit](#omit)|Same as `@stubgen:omit` | | Yes| No (but has side-effects)| Class, Method|
+|[@stubgen:include](#stubgen_include)|Insert another C++ file ||Yes| Yes|File|
+|[@stubgen:stub](#stubgen_stub)|Emit empty function stub instead of full proxy implementation | | Yes| No| Method|
+|[@stub](#stub)|Same as `@stubgen:stub` | | Yes| No|Method|
+|[@insert](#insert)|Same as `@stubgen:include` | | Yes|Yes|File|
+|[@define](#define)|Defines a literal as a known identifier | | Yes |Yes|File|
+
+<hr/>
+
+#### @stubgen:skip
+!!! warning
```

Comment:
Perhaps instead of this !!! exclamation style use the markup's blockquoute `>` ?

Reply:
The `!!!` syntax generates admonitions (https://squidfunk.github.io/mkdocs-material/reference/admonitions/) when the documentation site is built, so is preferable to just standard markdown blockquotes for doing callouts